components/developer/rust/Makefile

```diff
@@ -184,6 +194,8 @@ install:	$(INSTALL_64)
 REQUIRED_PACKAGES += utility/bash-completion
 REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
+REQUIRED_PACKAGES += developer/gcc-7
```

Comment:
Please, remove developer/gcc-7, it's part of build-essential now

Reply:
Done